### PR TITLE
Active customer

### PR DESCRIPTION
**Pull request by mouniamekrane**
working on active customer

### CODE REVIEW

File: src/Footer/footer.js [hunk 1]
```diff
@@ -3,10 +3,259 @@ import { CountActive } from './ActiveClients/activeClients.js'
 import { RowsPerPageClients } from './RowsPerPage/RowsPerPage.js'
 import { ClientsOfPage } from './PageOf/ClientsOf.js'
 import { NextPageClient } from './Pagination/nextPage.js'
+const clients = [{
```
Comment (faouzioudouh) [inline src/Footer/footer.js:6]: Why are you copying cutomers here?

File: src/Footer/ActiveClients/activeClients.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
+
 import './activeClients.css'
-export const CountActive = () => {
-    return (
-        <div className="clients" id="actived-clients">ACTIVE CUSTOMERS :</div>
-    )
-}
\ No newline at end of file
+export const CountActive = ({ clientsToCount }) => {
```
Comment (faouzioudouh) [inline src/Footer/ActiveClients/activeClients.js:3]: What is this? component? function?

**Review by faouzioudouh — CHANGES_REQUESTED** [review]
You are missing a lot, I couldn't understand your code or what you are trying to achieve :)

File: src/Footer/ActiveClients/activeClients.js (hunk 2 above)
Comment (mouniamekrane) [reply src/Footer/ActiveClients/activeClients.js:3]: its a component

File: src/Footer/footer.js (hunk 1 above)
Comment (mouniamekrane) [reply src/Footer/footer.js:6]: i did a mistake here

**Pull request merged**